README.md

```diff
@@ -11,14 +11,16 @@ for any bug bounty. We advise you to be careful and experiment on the network at
 
 ## Cmd usage
 
-The latest release version is compatible with the greenfield. If the user need to use the command tool on the test net of greenfield, he will need to switch to the latest release version.
+Please notice that the latest release version is compatible with the greenfield system of test net. 
```

Comment:
Greenfield is still undergoing rapid development iterations, and greenfield-cmd also needs to be continuously updated and adapted. When using it, please do not directly use the master branch or develop branch. If you are using this tool on the Greenfield Testnet, please switch to the latest official release version. To obtain the latest version of the command tool, please visit the following URL: https://github.com/bnb-chain/greenfield-cmd/releases.

Reply:
fixed